enhancements/distributed-tracing/distributed-tracing.md

```diff
@@ -0,0 +1,119 @@
+---
+title: distributed-tracing-with-opentelemetry
+authors:
+  - "@sallyom"
+  - "@parulsingh"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-04-14
+last-updated: 2021-06-09
+status: informational
+---
+
+# Distributed Tracing with OpenTelemetry
```

Comment:
Is this only about instrumenting the components that are part of OpenShift? Or about adding distributed tracing capabilities in general? I'm asking because [kspan](https://github.com/weaveworks-experiments/kspan) has been mentioned as a nice tool to visualize Kubernetes events, but wouldn't be covered by this proposal here, as all it requires is a backend to send data to (Jaeger, for instance).

Reply:
I will be looking into kspan, I'll report back. As of now, it's only about instrumenting the components and/or turning on tracing for upstream components. If it makes sense, we can make it more broad and include use of kspan.